timelock-server/src/integTest/java/com/palantir/atlasdb/timelock/TimeLockTestUtils.java

```diff
@@ -51,7 +56,7 @@ static TransactionManager createTransactionManager(TestableTimelockCluster clust
                 .build();
         return TransactionManagers.builder()
                 .config(config)
-                .userAgent("test")
```

Comment:
I don't think this was meant to be connected with the `"test"` for the namespace, but sure